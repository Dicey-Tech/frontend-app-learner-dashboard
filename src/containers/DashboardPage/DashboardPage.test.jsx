'use strict';

import React from 'react';
import renderer from 'react-test-renderer';
import { DashboardPage } from '.';

it('dashboard renders correctly', () => {
  const tree = renderer.create(<DashboardPage />).toJSON();
  expect(tree).toMatchSnapshot();
});
