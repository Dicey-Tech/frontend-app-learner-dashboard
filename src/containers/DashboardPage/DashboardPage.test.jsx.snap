// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dashboard renders correctly 1`] = `
<main>
  <div
    className="container-fluid"
  >
    <div
      className="p-4"
    >
      <div>
        <span
          className="sectionTitle"
        >
          My Classes
        </span>
        <div
          className="row p-3 mb-2"
        >
          <div
            className="row card-deck"
          >
            <div
              className="exploreCard card"
            >
              <div
                className="exploreCardBody card-body"
              >
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-plus-circle fa-w-16 card-icon"
                  data-icon="plus-circle"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 512 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
                <p
                  className="card-text"
                >
                  New Class
                </p>
              </div>
            </div>
            <div
              className="courseCard card"
            >
              <img
                className="card-img-top card-image"
                src="test-file-stub"
              />
              <div
                className="pt-2 pl-4 pr-4 pb-2 position-relative card-body"
              >
                <div
                  className="row"
                >
                  <div
                    className="card-title h5"
                  >
                    Course-1
                  </div>
                </div>
                <div
                  className="row"
                >
                  <div
                    className="col col-6 p-0"
                  >
                    <div
                      className="card-subtitle h6"
                    >
                      Age: 12-15
                    </div>
                  </div>
                  <div
                    className="col col-6 p-0 text-right"
                  >
                    <div
                      className="card-subtitle h6"
                    >
                      25 hours
                    </div>
                  </div>
                </div>
                <div
                  className="row"
                >
                  <p
                    className="card-description card-text"
                  >
                    This course teaches students how to create a 3D printed model
                  </p>
                </div>
                <div
                  className="row"
                  id="action-row"
                >
                  <div
                    className="col col-6 p-0"
                  >
                    <button
                      className="btn btn-primary"
                      disabled={false}
                      type="button"
                    >
                      Start Now
                    </button>
                  </div>
                  <div
                    className="col col-6 p-0 text-right"
                  >
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-bookmark fa-w-12 bookmark-icon"
                      data-icon="bookmark"
                      data-prefix="far"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 384 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M336 0H48C21.49 0 0 21.49 0 48v464l192-112 192 112V48c0-26.51-21.49-48-48-48zm0 428.43l-144-84-144 84V54a6 6 0 0 1 6-6h276c3.314 0 6 2.683 6 5.996V428.43z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                  </div>
                </div>
              </div>
            </div>
            <div
              className="courseCard card"
            >
              <img
                className="card-img-top card-image"
                src="test-file-stub"
              />
              <div
                className="pt-2 pl-4 pr-4 pb-2 position-relative card-body"
              >
                <div
                  className="row"
                >
                  <div
                    className="card-title h5"
                  >
                    Course-2
                  </div>
                </div>
                <div
                  className="row"
                >
                  <div
                    className="col col-6 p-0"
                  >
                    <div
                      className="card-subtitle h6"
                    >
                      Age: 12-15
                    </div>
                  </div>
                  <div
                    className="col col-6 p-0 text-right"
                  >
                    <div
                      className="card-subtitle h6"
                    >
                      25 hours
                    </div>
                  </div>
                </div>
                <div
                  className="row"
                >
                  <p
                    className="card-description card-text"
                  >
                    This course teaches students how to create a 3D printed model
                  </p>
                </div>
                <div
                  className="row"
                  id="action-row"
                >
                  <div
                    className="col col-6 p-0"
                  >
                    <button
                      className="btn btn-primary"
                      disabled={false}
                      type="button"
                    >
                      Start Now
                    </button>
                  </div>
                  <div
                    className="col col-6 p-0 text-right"
                  >
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-bookmark fa-w-12 bookmark-icon"
                      data-icon="bookmark"
                      data-prefix="far"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 384 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M336 0H48C21.49 0 0 21.49 0 48v464l192-112 192 112V48c0-26.51-21.49-48-48-48zm0 428.43l-144-84-144 84V54a6 6 0 0 1 6-6h276c3.314 0 6 2.683 6 5.996V428.43z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div>
        <span
          className="sectionTitle"
        >
          My Courses
        </span>
        <div
          className="row p-3 mb-2"
        >
          <div
            className="row card-deck"
          >
            <div
              className="exploreCard card"
            >
              <div
                className="exploreCardBody card-body"
              >
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-search-plus fa-w-16 card-icon"
                  data-icon="search-plus"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 512 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M304 192v32c0 6.6-5.4 12-12 12h-56v56c0 6.6-5.4 12-12 12h-32c-6.6 0-12-5.4-12-12v-56h-56c-6.6 0-12-5.4-12-12v-32c0-6.6 5.4-12 12-12h56v-56c0-6.6 5.4-12 12-12h32c6.6 0 12 5.4 12 12v56h56c6.6 0 12 5.4 12 12zm201 284.7L476.7 505c-9.4 9.4-24.6 9.4-33.9 0L343 405.3c-4.5-4.5-7-10.6-7-17V372c-35.3 27.6-79.7 44-128 44C93.1 416 0 322.9 0 208S93.1 0 208 0s208 93.1 208 208c0 48.3-16.4 92.7-44 128h16.3c6.4 0 12.5 2.5 17 7l99.7 99.7c9.3 9.4 9.3 24.6 0 34zM344 208c0-75.2-60.8-136-136-136S72 132.8 72 208s60.8 136 136 136 136-60.8 136-136z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
                <p
                  className="card-text"
                >
                  Find new courses
                </p>
              </div>
            </div>
            <div
              className="courseCard card"
            >
              <img
                className="card-img-top card-image"
                src="test-file-stub"
              />
              <div
                className="pt-2 pl-4 pr-4 pb-2 position-relative card-body"
              >
                <div
                  className="row"
                >
                  <div
                    className="card-title h5"
                  >
                    Course-1
                  </div>
                </div>
                <div
                  className="row"
                >
                  <div
                    className="col col-6 p-0"
                  >
                    <div
                      className="card-subtitle h6"
                    >
                      Age: 12-15
                    </div>
                  </div>
                  <div
                    className="col col-6 p-0 text-right"
                  >
                    <div
                      className="card-subtitle h6"
                    >
                      25 hours
                    </div>
                  </div>
                </div>
                <div
                  className="row"
                >
                  <p
                    className="card-description card-text"
                  >
                    This course teaches students how to create a 3D printed model
                  </p>
                </div>
                <div
                  className="row"
                  id="action-row"
                >
                  <div
                    className="col col-6 p-0"
                  >
                    <button
                      className="btn btn-primary"
                      disabled={false}
                      type="button"
                    >
                      Start Now
                    </button>
                  </div>
                  <div
                    className="col col-6 p-0 text-right"
                  >
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-bookmark fa-w-12 bookmark-icon"
                      data-icon="bookmark"
                      data-prefix="far"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 384 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M336 0H48C21.49 0 0 21.49 0 48v464l192-112 192 112V48c0-26.51-21.49-48-48-48zm0 428.43l-144-84-144 84V54a6 6 0 0 1 6-6h276c3.314 0 6 2.683 6 5.996V428.43z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                  </div>
                </div>
              </div>
            </div>
            <div
              className="courseCard card"
            >
              <img
                className="card-img-top card-image"
                src="test-file-stub"
              />
              <div
                className="pt-2 pl-4 pr-4 pb-2 position-relative card-body"
              >
                <div
                  className="row"
                >
                  <div
                    className="card-title h5"
                  >
                    Course-2
                  </div>
                </div>
                <div
                  className="row"
                >
                  <div
                    className="col col-6 p-0"
                  >
                    <div
                      className="card-subtitle h6"
                    >
                      Age: 12-15
                    </div>
                  </div>
                  <div
                    className="col col-6 p-0 text-right"
                  >
                    <div
                      className="card-subtitle h6"
                    >
                      25 hours
                    </div>
                  </div>
                </div>
                <div
                  className="row"
                >
                  <p
                    className="card-description card-text"
                  >
                    This course teaches students how to create a 3D printed model
                  </p>
                </div>
                <div
                  className="row"
                  id="action-row"
                >
                  <div
                    className="col col-6 p-0"
                  >
                    <button
                      className="btn btn-primary"
                      disabled={false}
                      type="button"
                    >
                      Start Now
                    </button>
                  </div>
                  <div
                    className="col col-6 p-0 text-right"
                  >
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-bookmark fa-w-12 bookmark-icon"
                      data-icon="bookmark"
                      data-prefix="far"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 384 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M336 0H48C21.49 0 0 21.49 0 48v464l192-112 192 112V48c0-26.51-21.49-48-48-48zm0 428.43l-144-84-144 84V54a6 6 0 0 1 6-6h276c3.314 0 6 2.683 6 5.996V428.43z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div>
        <span
          className="sectionTitle"
        >
          My Bookmarks
        </span>
        <div
          className="row p-3 mb-2"
        >
          <div
            className="row card-deck"
          >
            <div
              className="exploreCard card"
            >
              <div
                className="exploreCardBody card-body"
              >
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-search-plus fa-w-16 card-icon"
                  data-icon="search-plus"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 512 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M304 192v32c0 6.6-5.4 12-12 12h-56v56c0 6.6-5.4 12-12 12h-32c-6.6 0-12-5.4-12-12v-56h-56c-6.6 0-12-5.4-12-12v-32c0-6.6 5.4-12 12-12h56v-56c0-6.6 5.4-12 12-12h32c6.6 0 12 5.4 12 12v56h56c6.6 0 12 5.4 12 12zm201 284.7L476.7 505c-9.4 9.4-24.6 9.4-33.9 0L343 405.3c-4.5-4.5-7-10.6-7-17V372c-35.3 27.6-79.7 44-128 44C93.1 416 0 322.9 0 208S93.1 0 208 0s208 93.1 208 208c0 48.3-16.4 92.7-44 128h16.3c6.4 0 12.5 2.5 17 7l99.7 99.7c9.3 9.4 9.3 24.6 0 34zM344 208c0-75.2-60.8-136-136-136S72 132.8 72 208s60.8 136 136 136 136-60.8 136-136z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
                <p
                  className="card-text"
                >
                  Find new courses
                </p>
              </div>
            </div>
            <div
              className="courseCard card"
            >
              <img
                className="card-img-top card-image"
                src="test-file-stub"
              />
              <div
                className="pt-2 pl-4 pr-4 pb-2 position-relative card-body"
              >
                <div
                  className="row"
                >
                  <div
                    className="card-title h5"
                  >
                    Course-1
                  </div>
                </div>
                <div
                  className="row"
                >
                  <div
                    className="col col-6 p-0"
                  >
                    <div
                      className="card-subtitle h6"
                    >
                      Age: 12-15
                    </div>
                  </div>
                  <div
                    className="col col-6 p-0 text-right"
                  >
                    <div
                      className="card-subtitle h6"
                    >
                      25 hours
                    </div>
                  </div>
                </div>
                <div
                  className="row"
                >
                  <p
                    className="card-description card-text"
                  >
                    This course teaches students how to create a 3D printed model
                  </p>
                </div>
                <div
                  className="row"
                  id="action-row"
                >
                  <div
                    className="col col-6 p-0"
                  >
                    <button
                      className="btn btn-primary"
                      disabled={false}
                      type="button"
                    >
                      Start Now
                    </button>
                  </div>
                  <div
                    className="col col-6 p-0 text-right"
                  >
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-bookmark fa-w-12 bookmark-icon"
                      data-icon="bookmark"
                      data-prefix="far"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 384 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M336 0H48C21.49 0 0 21.49 0 48v464l192-112 192 112V48c0-26.51-21.49-48-48-48zm0 428.43l-144-84-144 84V54a6 6 0 0 1 6-6h276c3.314 0 6 2.683 6 5.996V428.43z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                  </div>
                </div>
              </div>
            </div>
            <div
              className="courseCard card"
            >
              <img
                className="card-img-top card-image"
                src="test-file-stub"
              />
              <div
                className="pt-2 pl-4 pr-4 pb-2 position-relative card-body"
              >
                <div
                  className="row"
                >
                  <div
                    className="card-title h5"
                  >
                    Course-2
                  </div>
                </div>
                <div
                  className="row"
                >
                  <div
                    className="col col-6 p-0"
                  >
                    <div
                      className="card-subtitle h6"
                    >
                      Age: 12-15
                    </div>
                  </div>
                  <div
                    className="col col-6 p-0 text-right"
                  >
                    <div
                      className="card-subtitle h6"
                    >
                      25 hours
                    </div>
                  </div>
                </div>
                <div
                  className="row"
                >
                  <p
                    className="card-description card-text"
                  >
                    This course teaches students how to create a 3D printed model
                  </p>
                </div>
                <div
                  className="row"
                  id="action-row"
                >
                  <div
                    className="col col-6 p-0"
                  >
                    <button
                      className="btn btn-primary"
                      disabled={false}
                      type="button"
                    >
                      Start Now
                    </button>
                  </div>
                  <div
                    className="col col-6 p-0 text-right"
                  >
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-bookmark fa-w-12 bookmark-icon"
                      data-icon="bookmark"
                      data-prefix="far"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 384 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M336 0H48C21.49 0 0 21.49 0 48v464l192-112 192 112V48c0-26.51-21.49-48-48-48zm0 428.43l-144-84-144 84V54a6 6 0 0 1 6-6h276c3.314 0 6 2.683 6 5.996V428.43z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</main>
`;
